linera-views/src/common.rs

```diff
@@ -41,6 +42,16 @@ pub(crate) enum Update<T> {
 /// The minimum value for the view tags. Values in 0..MIN_VIEW_TAG are used for other purposes.
 pub(crate) const MIN_VIEW_TAG: u8 = 1;
 
+/// Data type indicating that the database is not coherent
+#[derive(Error, Debug)]
+pub struct DatabaseCoherencyError(String);
```

Comment:
Please use `thiserror` and create a proper enum with a variant for each error case. (See e.g. linera_execution::Error)

Reply:
Ok.